bindings/core/src/method_handler/secret_manager.rs

```diff
@@ -135,6 +135,18 @@ where
                 return Err(iota_sdk::client::Error::SecretManagerMismatch.into());
             }
         }
+        #[cfg(feature = "stronghold")]
+        SecretManagerMethod::SetStrongholdPassword { password } => {
+            if let Some(secret_manager) = secret_manager.downcast::<StrongholdSecretManager>() {
+                secret_manager.set_password(password).await?;
+                Response::Ok
+            } else if let Some(SecretManager::Stronghold(secret_manager)) = secret_manager.downcast::<SecretManager>() {
+                secret_manager.set_password(password).await?;
+                Response::Ok
+            } else {
+                return Err(iota_sdk::client::Error::SecretManagerMismatch.into());
+            }
```

Comment:
I think this also works
```suggestion
            let stronghold = if let Some(secret_manager) = secret_manager.downcast::<StrongholdSecretManager>() {
                secret_manager
            } else if let Some(SecretManager::Stronghold(secret_manager)) = secret_manager.downcast::<SecretManager>() {
                secret_manager
            } else {
                return Err(iota_sdk::client::Error::SecretManagerMismatch.into());
            };
            stronghold.set_password(password).await?;
            Response::Ok
```

Reply:
Yeah, I can do it if you want. I just followed the same design as the method above ([store_mnemonic](https://github.com/iotaledger/iota-sdk/blob/d57dad9a35a7a169b119c65d17a2d0bee7712760/bindings/core/src/method_handler/secret_manager.rs#L126))

Reply:
Could also be changed, but it's also not that important